server/src/main/java/org/elasticsearch/repositories/RepositoriesModule.java

```diff
@@ -42,7 +42,7 @@ public final class RepositoriesModule {
     public static final String METRIC_UNSUCCESSFUL_OPERATIONS_COUNT = "es.repositories.operations.unsuccessful.count";
     public static final String METRIC_EXCEPTIONS_HISTOGRAM = "es.repositories.exceptions.histogram";
     public static final String METRIC_THROTTLES_HISTOGRAM = "es.repositories.throttles.histogram";
-
+    public static final String HTTP_REQUEST_TIME_IN_MICROS_HISTOGRAM = "es.repositories.requests.http_request_time.histogram";
```

Comment:
I wonder if we need to repeat request twice? I think this could be shortened to:
`es.repositories.requests.duration.histogram`